analysis/utility.py

```diff
@@ -0,0 +1,53 @@
+#/usr/bin/python 
+# npagane | python utilities file
+
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```

Reply:
thank you for all of these!! i really appreciate the guidance